auto_round/compressors/base.py

```diff
@@ -234,6 +234,9 @@ def __init__(
         device = kwargs.pop("device", None)
         # Scale factor for RAM usage per parameter.
         mem_per_param_scale = kwargs.pop("mem_per_param_scale", None)
+
+        if os.getenv("MODEL_PLATFORM"):
```

Comment:
Suggest follow VLLM usage, which is a bool flag VLLM_USE_MODELSCOPE. By default model platform is HF hub. You may use AUTOROUND_USE_MODELSCOPE or just USE_MODELSCOPE